avocado_vt/plugins/vt.py

```diff
@@ -163,18 +162,18 @@ def configure(self, parser):
         supported_uris = ", ".join(SUPPORTED_LIBVIRT_URIS)
         msg = ("Choose test connect uri for libvirt (E.g: %s). "
                "Current: %%(default)s" % supported_uris)
-        uri_current = settings.get_value('vt.libvirt', 'connect_uri',
+        uri_current = get_settings_value('vt.libvirt', 'connect_uri',
                                          default=None)
         vt_compat_group_libvirt.add_argument("--vt-connect-uri",
                                              action="store",
-                                             dest="vt_connect_uri",
+                                             dest="vt.libvirt.connect_uri",
                                              default=uri_current,
                                              help=msg)
 
-    def run(self, args):
+    def run(self, config):
         """
         Run test modules or simple tests.
 
-        :param args: Command line args received from the run subparser.
+        :param config: Command line args received from the run subparser.
```

Comment:
Do we also need to consider update the plugin `vt_list.py` ?
```suggestion
    def run(self, config):
        """
        Run test modules or simple tests.

        :config args: Command line args received from the run subparser.
        """
        loader.register_plugin(VirtTestLoader)
```